docs/installation_guide.rst

```diff
@@ -174,24 +188,37 @@ Download the executable installer for Windows available on the Python Software F
 
 .. _downloads page: https://www.python.org/downloads/
 
-2. Install Scout Python package
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A virtual environment should be used to ensure that package dependencies are isolated for Scout. Common tools for managing virtual environments include `venv`_, `virtualenv`_, and `conda`_. Begin by `opening a command prompt`_ window. To create and activate a virtual environment named ``.scout_env`` using the standard Python module, ``venv``, navigate to the Scout installation directory and run the following commands::
 
-Once Python 3 installation is complete, the Scout package and its dependencies can be installed. pip [#f1]_ is the tool you will use to install add-ons specific to Python 3. Begin by `opening a command prompt`_ window. We recommend using a virtual environment, such as ``venv``, ``vitualenv``, or ``conda`` to run Scout. Create and activate an environment and run the following from your Scout installation directory to install the Scout package::
+   py -3 -m venv .scout_env
+   .scout_env\Scripts\activate
 
-   py -3 -m pip install .
+This will activate the new Python environment where the Scout package will be installed. To check that the environment was set up correctly, run ``where python``, and ensure that the first path returned points to the new environment directory.
 
 .. note::
-   For developers: if you intend on editing files within the Scout package directory, such as ``scout/supporting_data`` or .py modules, run ``py -3 -m pip install -e ".[dev]"`` to install in editable mode with developer depedendencies.
+   Each time you intend on running Scout, the virtual environment must be activated.
 
 .. _Open a command prompt:
 .. _opening a command prompt: http://www.digitalcitizen.life/7-ways-launch-command-prompt-windows-7-windows-8
+.. _venv: https://docs.python.org/3/library/venv.html
+.. _virtualenv: https://virtualenv.pypa.io/en/latest/
+.. _conda: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html
+
+2. Install Scout Python package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once Python 3 installation is complete and a virtual environment is activated, the Scout package and its dependencies can be installed. pip [#f1]_ is the tool you will use to install add-ons specific to Python 3. Run the following from your Scout installation directory to install the Scout package::
+
+   python -m pip install .
```

Comment:
Note: with the assumption that a virtual environment is used, `python` is the command to point to the correct version inside the virtual env. Even without an environment, `python` should still work for Windows users if it was added to PATH at install. I updated to use `python` instead of `py -3` throughout the docs.